io-engine/src/pool_backend.rs

```diff
@@ -22,6 +26,7 @@ impl TryFrom<i32> for PoolBackend {
     fn try_from(value: i32) -> Result<Self, Self::Error> {
```

Comment:
Can we not use concrete types here and convert from/to_i32 ?

Reply:
Done